js/schedule.js

```diff
@@ -347,13 +370,28 @@ function Schedule(CUSTOM_CONFIG) {
     $('#session-detail-wrapper').remove();
   }
 
+  schedule.saveScrollState = function(page, scrollY) {
```

Comment:
no prob! 👍 I think there are 2 more places that need the check. I've left inline comments below.